### PR TITLE
Remove krabs.cpp reference and add missing header

### LINKED ISSUE #133: krabsetw.targets still uses krabs.cpp
Despite being removed in #96.

### PR DESCRIPTION
**Pull request by pravic**
closes #133 

Right now the [upstream version](https://www.nuget.org/packages/Microsoft.O365.Security.Krabsetw/4.1.12) is broken.

### CODE REVIEW

**Comment by pravic** [comment]
@swannman could you review this, please?

File: krabsetw.nuspec [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (swannman) [inline krabsetw.nuspec:1]: could you bump the <version> tag to `4.1.13` so we can publish an updated nuget? thanks!

**Comment by swannman** [comment]
Thanks for fixing this. If you bump the krabsetw nuget version I'll merge this in.

**Comment by pravic** [comment]
Done. Thanks!

**Pull request merged**